app/src/main/AndroidManifest.xml

```diff
@@ -15,20 +16,19 @@
     android:theme="@style/OppiaTheme">
     <activity
       android:name=".home.continueplaying.ContinuePlayingActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar"
-      android:screenOrientation="portrait" />
```

Comment:
Please re-add. We don't support landscape anywhere in the app currently.

Reply:
Re-added.